File: Lodjinha/app/src/main/java/br/com/bsavoini/lodjinha/api/model/BannersResponse.java
package br.com.bsavoini.lodjinha.api.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class BannersResponse {
    @SerializedName("data")
    private  List<BannerModel> bannersArr;

    public List<BannerModel> getBannersArr() {
        return bannersArr;
    }
}

File: model/src/main/java/alodjinha/cfgdemelo/com/model/BannersResponse.kt
package alodjinha.cfgdemelo.com.model

import com.google.gson.annotations.SerializedName

data class BannersResponse (
    @SerializedName("data")
    var banners: List<BannerContent>
)

data class BannerContent (
    @SerializedName("id")
    var id: Int,
    @SerializedName("linkUrl")
    var linkUrl: String,
    @SerializedName("urlImagem")
    var imageUrl: String
)